### PR TITLE
Resolves large trend issue

### PR DESCRIPTION
**Pull request by mshodge**
Resolves large trend issue #5 by sorting graphs in blocks (pages).

### CODE REVIEW

File: mobius/graphs.py [hunk 1]
```diff
@@ -40,6 +42,87 @@ def clear_buffer(num, path_buffer):
...
         return num, path_buffer, state
 
+    def convert_list_to_dict(lst):
+        """Converts a list (with lists in) to a dictionary"""
+        new_dict = {}
+        num = 1
+        for item in lst:
+            new_dict[num] = item
+            num += 1
+        return new_dict
```
Comment (philip-lee-ons) [inline mobius/graphs.py:52]:
```suggestion
        return dict(enumerate(lst, start=1))
```

File: mobius/graphs.py [hunk 2]
```diff
@@ -40,6 +42,87 @@ def clear_buffer(num, path_buffer):
...
+
+                graph_count = 0
+                order_values = []
+                original_order = list(range(0, 12))
+
+                for num in range(len(block)):
+                    """Rough way of getting block graphs in order. Scale y value larger as it is the most important"""
+                    graph_count += 1
+                    order_value = (round(float(str(block[num][2][0].start).split('+')[-1].split('j')[0]), -2) * 1000) + \
+                                  float((str(block[num][2][0].start).split('+')[0].split('(')[-1]))
```
Comment (philip-lee-ons) [inline mobius/graphs.py:88]: Just a quick suggestion:

```suggestion
                    order_path = block[num][2][0]
                    order_value = round(order_path.start.imag, -2) * 1000 + order_path.start.real                   
```

File: mobius/graphs.py [hunk 3]
```diff
@@ -40,6 +42,87 @@ def clear_buffer(num, path_buffer):
...
+                    shutil.copyfile(f"{output_folder}/svg/{file}", f"{output_folder}/tmp/{new_file}")
+                    num += 1
+
+            shutil.rmtree(f"{output_folder}/svg")
+            os.rename(f"{output_folder}/tmp", f"{output_folder}/svg")
+
+        """Convert list to dictionary"""
+        output_order = convert_list_to_dict(output_order)
+
+        return output_order
```
Comment (philip-lee-ons) [inline mobius/graphs.py:118]: Think this needs to be:

```suggestion
        reordered_output = dict((output_order[key], value) for key, value in output.items())
        return reordered_output
```
to preserve the output type of the function dict[int, list[Path]]

Comment (philip-lee-ons) [reply mobius/graphs.py:116]: ignore this, think the problem is further up

Comment (mshodge) [reply mobius/graphs.py:116]: Shouldn't be needed with the latest commits

**Pull request merged**